client/src/common/components/GridList.jsx

```diff
@@ -22,6 +22,7 @@ import Preloader from 'common/components/Preloader';
 class GridList extends PureComponent {
   handleFavClick = (itemId, isFavourite) => event => {
     event.stopPropagation();
+
```

Comment:
why all these new lines? Are they specific to this PR?

Reply:
Not specific, I just thought it looks cleaner to have a line of a break after a function call

Reply:
Please roll back as it mess up git history. If you want to make some code cleanup for not relevant files then create separate branch.